Dockerfile

```diff
@@ -4,14 +4,14 @@ FROM python:$PYTHON_VERSION
 USER root
 WORKDIR /opt
 RUN wget -c --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/8u131-b11/d54c1d3a095b4ff2b6607d096fa80163/jdk-8u131-linux-x64.tar.gz && \
-    wget https://downloads.lightbend.com/scala/2.13.5/scala-2.13.5.tgz && \
-    wget https://archive.apache.org/dist/spark/spark-3.1.1/spark-3.1.1-bin-hadoop3.2.tgz && \
+    wget https://downloads.lightbend.com/scala/2.12.15/scala-2.12.15.tgz && \
```

Comment:
Are we downgrading version of scala?

Reply:
There are inconsistent versions of scala in our codebase. I was just making them consistent. I am happy to bump it up to 2.13.x if we want to do that @svishal9 @sambhav13

Reply:
I would vote for keeping Scala version as `2.13.x` since that's the version used for Spark `3.x`